vita3k/ngs/include/ngs/mempool.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
```

Comment:
Oh missed this one, can you do this?

Reply:
Ahh, resolved it because of https://github.com/Vita3K/Vita3K/pull/657#discussion_r441965601. If you really want me to do it I'll do it :D Won't be hard- just might have limited use.

Reply:
I see. I looked around little bit and confirmed that ngs voice definition is really exceptional case. I really don't see why they made such an exception here. :0

Reply:
I'm puzzled as to why they need to be allocated at all. VoiceDefinitions look opaque, so I suppose we'd be able to just return some id instead of allocating anything. I remember looking into switching over to some id system and giving up for some reason 😖

Reply:
Yes, I also think we could just use id system here. It looks okay though. It wouldn't cause any trouble other than using little more guest memory.